### ISSUE TITLE
Multiparous

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10122]
**Preferred term label:**
Multiparous

**Synonyms**
Multiparity

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0026751

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Obturator hernia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000054
- label: Multiparity
- parents: [Past obstetric history (HP:0032467)](https://hpo.jax.org/app/browse/term/HP:0032467)
- pmid: 
- def: History of multiple previous births.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10122
- synonyms:
